### PR TITLE
Include and load leaf html in json data instead of using fetch

### PR DESCRIPTION
**Pull request by rlskoeser**
feedback during the code review process helped me see that we could include just the portion of the html we needed in the json data (rendering just the inner article portion with hugo), and not have to deal with fetching and parsing the content remotely.

### CODE REVIEW

**Comment by rlskoeser** [comment]
@quadrismegistus do you have time for a quick review? mostly care about your eyes on the js in case there's anything I missed

File: assets/js/panel.js [hunk 1]
```diff
@@ -104,46 +104,15 @@ class Panel {
...
-        }
-        this.el.querySelector("article").replaceWith(article);
-        // store loaded url in data attribute for reload check
-        this.el.dataset.showing = url;
-      })
-      .catch((response) => {
-        // if the request failed, display error article
-        let errorArticle = document.querySelector("#loaderror").cloneNode(true);
-        this.el.querySelector("article").replaceWith(errorArticle);
-      });
```
Comment (quadrismegistus) [inline assets/js/panel.js:146]: We don't need this anymore because we're no longer fetching URLs in the same way? Or are we somehow bypassing fetching?

**Review by quadrismegistus — APPROVED** [review]
Looks great! Seems like great improvement if it bypasses more requests

File: assets/js/panel.js (hunk 1 above)
Comment (rlskoeser) [reply assets/js/panel.js:146]: we're bypassing the fetch entirely - I'm including just the html we need in the json data we generate by hugo

Comment (quadrismegistus) [reply assets/js/panel.js:146]: Clever! nice

**Pull request merged**